sdk/client/src/blocking.rs

```diff
@@ -260,6 +262,7 @@ impl BlockingClient {
     fn send_impl<S: Serialize, T: DeserializeOwned>(&self, payload: &S) -> Result<T> {
         let mut request = ureq::post(&self.url)
             .timeout_connect(REQUEST_TIMEOUT)
+            .set(reqwest::header::USER_AGENT.as_ref(), USER_AGENT)
```

Comment:
Unfortunately you're not going to be able to do this since the `reqwest` dependency is optional and is not enabled when you only enable the blocking client. So in this case you'll have to continue using a hardcoded `"User-Agent"` string.